metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/exceptionhandling/ErrorMessage.java

```diff
@@ -22,21 +22,20 @@
 import java.util.Date;
 
 public class ErrorMessage {
-    private long timestamp;
 
-    private int status;
+    public final static String SAMPLE_WITHOUT_TAXONOMY = "A sample must have atleast one taxonomy";
```

Comment:
Space between "at" and "least". Same below.

Reply:
depends on dictionary :) changed it